assignments/week_4/week_4_exercises/exercise_4_1/exercise_4_1.md

```diff
@@ -0,0 +1,127 @@
+# Exercise 4.1
+This is the first exercise of week 4, and the first one involving ROS with C++!
+
+# Exercise Objective
+For this exercise you will be adding a subscriber to a ROS node. The end result will be a node which
+subscribes to `/string_topic`, and prints the messages data to the ROS console.
+
+Adding a subscriber involves:
+ 
+ * Including the correct message header
+ * Making a callback function
+ * Creating the subscriber object
+ 
+To test that your subscriber works, you will:
+ * Run the node using `roslaunch`
```

Comment:
can we just do 2 rosrun commands since we have not covered roslaunch?

Reply:
Uh... this will make the sequencing of this more annoying actually... I guess it is fine?